packages/x-components/src/x-modules/next-queries/components/__tests__/next-query.spec.ts

```diff
@@ -5,7 +5,7 @@ import { getDataTestSelector, installNewXPlugin } from '../../../../__tests__/ut
 import NextQuery from '../next-query.vue';
 
 describe('testing next query item component', () => {
-  const suggestion = getSuggestionsStub('NextQuery')[0];
+  const suggestion = getSuggestionsStub()[0];
```

Comment:
You can use the `const nextQuerySuggestion = getNextQueriesStub()[0];`

Also you can remove the `query-suggestions-stubs.factory.ts` as now we're using the `suggestions-stubs.factory.ts`. After that yo need to remove  this file from the `index.ts` of the stubs and update the calls in `getter.spec.ts` of the query suggestions.

Finally I realised that the `related-tags-stubs.factory.ts` is giving an error because of the model name (it's `RelatedTag` not `RelatedTags`)

Reply:
I you need help, just tell me and we can do it together!

Reply:
As per external conversation, we'll leave it as it is since there is already another branch where these changes are being applied